src/scheduler/MpiWorld.cpp

```diff
@@ -123,6 +123,12 @@ void MpiWorld::create(faabric::Message& call, int newId, int newSize)
             msg.set_cmdline(thisRankMsg->cmdline());
             msg.set_inputdata(thisRankMsg->inputdata());
             msg.set_migrationcheckperiod(thisRankMsg->migrationcheckperiod());
+            // To run migration experiments easily, we may want to propagate
+            // the `UNDERFULL` topology hint. In general however, we don't
```

Comment:
We don't use markup syntax in inline comments like this as it'll never be rendered as markup.